tests/versions/apm_server.yml

```diff
@@ -1,7 +1,5 @@
 APM_SERVER:
-  - '6.x'
-  - '6.6;--release'
   - '6.5;--release'
-#  - '6.4;--release'
-#  - '6.3;--release'
-#  - '6.2;--release'
+  - '6.4;--release'
+  - '6.3;--release'
+  - '6.2;--release'
```

Comment:
APM Server `6.6` and `6.7` also support `v1`, they should be added.

Reply:
added, I have supposed that every agent that runs on 6.5 runs on 6.6 and 6.7